src/pybind/mgr/volumes/fs/fs_util.py

```diff
@@ -143,6 +143,25 @@ def listsnaps(fs, volspec, snapdirpath, filter_inherited_snaps=False):
         raise VolumeException(-e.args[0], e.args[1])
     return snaps
 
+def get_all_dir_entries(fs, dirpath):
```

Comment:
Why aren't we using `listdir()` with `filter_files=False` instead of a new method?